exporter/otlphttpexporter/config.go

```diff
@@ -14,9 +14,9 @@ import (
 
 // Config defines configuration for OTLP/HTTP exporter.
 type Config struct {
-	confighttp.HTTPClientSettings `mapstructure:",squash"`     // squash ensures fields are correctly decoded in embedded struct.
-	QueueConfig                   exporterhelper.QueueSettings `mapstructure:"sending_queue"`
-	RetryConfig                   configretry.BackOffConfig    `mapstructure:"retry_on_failure"`
+	confighttp.HTTPClientConfig `mapstructure:",squash"`     // squash ensures fields are correctly decoded in embedded struct.
```

Comment:
This is a breaking change. the old field should be kept as deprecated

Reply:
Actually. It'll be pretty hard to keep it backward-compatible. I think we should allow breaking change here. It's only applicable to the API, not the user interface tho. Please mark it accordingly in the changelog

Reply:
yes, I was looking for a way to do struct field aliasing or something like that, but no good solution seems to exist. This only breaks for programmatic use so mostly touching on tests in contrib. I will add to changelog, thanks!